### ISSUE TITLE
Upstream all manual updates

### ISSUE THREAD

**Issue by leamas** [opencpn-manuals/plugins#23]
Currently, here is  a number of forked repos used for initial setup of the manual:

Plugin                                      |   PR
--------------------------|------------------------------
Deviation_pi | https://github.com/RooieDirk/Deviation_pi/pull/12

aisradar currently lives by nohal, but should be moved to the canonical Verezano upstream as soon as PR is accepted.

The ultimate goal is to transfer control to upstream for these repos. Workflow for each plugin:

1. Possibly update to standard config  (#22)
2. Make a PR to upstream
3. Wait for PR to be merged.
3. Update site.yml and sources.state to use the upstream repo
4. Remove the opencpn-manuals fork
5.  Remove entry in table above

We should have made a PR for all repos before  we announce the manual.

**Comment by leamas** [comment]
@rasbats: I have this feeling... Have you reviewed the other repos for similar problems?

**Comment by leamas** [comment]
Anyway: I remove the calculator fork and the line in  table above

**Comment by leamas** [comment]
vdr_pi is already done. Removing.

**Comment by Rasbats** [comment]
[https://github.com/opencpn-manuals/plugins/issues/22](https://github.com/opencpn-manuals/plugins/issues/22)

This was an attempt to summarise the PR needed, based on a review of site.yml. Your list is probably more useful. I will keep out of the PR to give you space for your work.

**Comment by leamas** [comment]
Please don't keep out of the PR business, I'm soo bored :)

Could you file for the repos you have worked with it would be great. This includes those which have a pending PR from oplaydo.

Talking of which. Some of the repos have an outdated (w r t the new structure) manual PR from oplaydo pending. Perhaps you just close this and make a new which goes directly to the new structure. Or just push some new commits on the PR branch, this is possible as long as the PR isn't merged.

**Comment by leamas** [comment]
BTW: I don't think my table is "more useful" than the list in #22.  We need the work in #22 before we can dive into that table safely. It's just different issues. 

I really, really appreciate how you organize things this way. Thanks!

**Comment by Rasbats** [comment]
Trying to work out how "partials" is used. Intend removing this folder unless you can see how it is used.

**Comment by leamas** [comment]
> this folder

?

**Comment by leamas** [comment]
@rasbats: Thanks!

**Comment by rgleason** [comment]
Merged. Statusbar. Sorry thought that was done.
statusbar_pi | rgleason/statusbar_pi#7
https://github.com/rgleason/statusbar_pi

**Comment by Rasbats** [comment]
@leamas , @rgleason 
I am sending an email to Steve Adler, asking if he can accept the PR for the 3 TwoCanPlugins.

**Comment by leamas** [comment]
Great!

BTW: Things will become cleaner if you avoid merging stuff. Basically, `git  rebase`and `git merge` does the same thing, but using rebase means a single line in history which is mush easier to track. It also has the benefit of `git rebase  --abort`  which is a nice escape door.

So, in short: Could you possibly try using `git rebase` instead of `git merge`before you push?

EDIT: Sorry, this was totally off-topic, my bad. Moved to https://github.com/opencpn-manuals/main/discussions/14#discussioncomment-1300939

**Comment by rgleason** [comment]
Could  we change to the AISradar  in  opencpn-manuals  for the time being?

**Comment by leamas** [comment]
@rgleason: Why did you close this?  There is still three plugins to handle.

**Comment by leamas** [comment]
Yes, but why?

**Comment by rgleason** [comment]
Very sorry, don't know.
